composer.json

```diff
@@ -24,7 +24,7 @@
     "php": "^7.3.0 || ^8.0.0",
     "ext-simplexml": "*",
     "nikic/php-parser": "^4.11.0",
-    "psr/container": "^1.1.1",
+    "psr/container": "^2.0.1",
```

Comment:
```suggestion
    "psr/container": "^1.1.1 || ^2.0.1",
```

Reply:
No. I won't support v1 and v2.

Reply:
Others allowed it though. It's literally no extra work. https://github.com/mezzio/mezzio-session/pull/27

Reply:
Tbh the only reason why it would be a problem is because of the container implementations in test suite. Maybe consider using anonymous classes? A v2 container is also a valid v1 container, a v1 container is not always a compatible v2 container.